CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
+# 3.0.0
+
+- move to typescript
+- add useImage hook
+- allow for an image loader to be injected
+- BREAKING: requires react 16.6 or higher
```

Comment:
hooks was officially introduced in 16.8

https://reactjs.org/docs/hooks-intro.html

```suggestion
- BREAKING: requires react 16.8 or higher
```